src/utils/utils.ts

```diff
@@ -275,6 +275,10 @@ export function bostedKommune(bruker: BrukerModell, geografiskbostedData) {
     return '-';
 }
 
+export const bostedBydelEllerUkjent = (bostedBydel: string, geografiskbostedData: Map<string, string>): string => {
+    return geografiskbostedData.get(bostedBydel) || 'Ukjent';
```

Comment:
Eg er ikkje sikker på om det vert rett å bruke "Ukjent" her. For kommune ser "ukjent" ut til å vere noko førehandsdefinert, her er det ikkje det.

Kanskje "Kunne ikke hente bydelsnavn" eller "-" ville vore betre?

Reply:
I de fleste kommuner så er det ingen bydeler å hente, da tenker jeg det kan bli overflod av "ukjent"-er, så jeg syns vi skal bruke `–`. Og også erstatte `||`, [SonarCloud har reagert på det](https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1125&id=navikt_veilarbportefoljeflatefs&open=AZLTqppN8kM1JeCcMvxM&tab=why), men viser det ikke her på GitHub fordi den syns dupliseringen er viktigere 🤷 
```suggestion
    return geografiskbostedData.get(bostedBydel) ?? '–';
```

Reply:
Slik det er no trur eg ein berre får "ukjent" dersom personen har ein bydel, men vi ikkje greier finne bydelsnamnet. Dersom dei ikkje har bydelsdata får dei "-" eit nivå opp i koden.

Er uansett einig, betre med ingen informasjon enn forvirrande informasjon, så går for "-" 😊